static/app/components/onboarding/docWithProductSelection.tsx

```diff
@@ -83,6 +147,24 @@ export function DocWithProductSelection({
     );
   }
 
+  const hasSessionReplay = organization.features.includes('session-replay');
+  const hasPerformance = organization.features.includes('performance-view');
+  const hasRightsToUgradePlan =
```

Comment:
this will be updated in a follow-up as I just confirmed that only owners can update plans.